src/main/java/uk/gov/hmcts/ccd/domain/service/callbacks/retrycontext/CallbackRetryContextBuilder.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.hmcts.ccd.domain.service.callbacks.retrycontext;
+
+import com.google.common.collect.Lists;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.ccd.ApplicationParams;
+
+import java.util.List;
+
+@Service
+public class CallbackRetryContextBuilder {
```

Comment:
can you add javadoc please? explaining the retry strategy

Reply:
Sure done now.